[1.4] Update Chinese localization

sgkoishi opened the pull request:
Update new texts.

Cyrillya committed:
Update tModLoader.json

Replace the names of some biomes with the translations from the Official Terraria Chinese Wiki
Add tags

sgkoishi committed:
Update Chinese localization

sgkoishi committed:
Update Chinese localization
Synchronize with latest en-US changes

JavidPack commented:
Can you update  "FirstLaunchWelcomeMessage" to the new version?

<https://github.com/tModLoader/tModLoader/blob/1.4/patches/tModLoader/Terraria/Localization/Content/en-US/tModLoader.json#L19>

Also, I think a few more keys were added and this patch now has merge conflicts, can you compare against the version above and merge your changes in to fix the conflicts?

sgkoishi committed:
Merge upstream

sgkoishi commented:
Yes, I saw those updates only exists in `en-US` but not in `zh-Hans` so I add the translated version manually, and it conflicts with the auto-synced version. However, there are some new texts added recently that were not synced:
https://github.com/tModLoader/tModLoader/commit/11ecd0b96a159f9af8fec968e4fceb7b58b95858#diff-158a57fd2a31ad91b0a13dea57bf9f8af290b50db641ea334247d93829ced7db
https://github.com/tModLoader/tModLoader/commit/44a529539740a5ac1abe64618328c364dfc10939#diff-158a57fd2a31ad91b0a13dea57bf9f8af290b50db641ea334247d93829ced7db

Should I just add them before those keys were added to zh-Hans?

sgkoishi committed:
Update Chinese localization
Synchronize with latest en-US changes

sgkoishi committed:
Fix typo

sgkoishi committed:
Merge PR 2480

Cyrillya committed:
Tooltip lines localization from PR 2479

Localized new tooltip lines from https://github.com/tModLoader/tModLoader/pull/2479

Crapsky233 commented:
https://github.com/chi-rei-den/tModLoader/pull/1 made a PR to add new texts from #2479

sgkoishi committed:
Merge branch 'Crapsky233/patch-3' into localization-zh-hans

sgkoishi committed:
Update Chinese localization
Unify wording, clarify common misconception, sync with en-US
